weather_mv/loader_pipeline/sinks.py

```diff
@@ -406,7 +409,19 @@ def open_dataset(uri: str,
     """Open the dataset at 'uri' and return a xarray.Dataset."""
     try:
         if is_zarr:
+            start_date = end_date = None
+            if 'start_date' in open_dataset_kwargs:
+                start_date = open_dataset_kwargs['start_date']
+                if open_dataset_kwargs is not None:
+                    del open_dataset_kwargs['start_date']
```

Comment:
Can you call something like `pop()` here?

Reply:
Thank you for suggestions.